### PR TITLE
chore: added rwanda location apis

### PR DESCRIPTION
**Pull request by micro-chipset**
- [ ] My submission is formatted according to the guidelines in the [contributing guide](/CONTRIBUTING.md)
- [ ] My addition is ordered alphabetically
- [ ] My submission has a useful description
- [ ] The description does not end with punctuation
- [ ] Each table column is padded with one space on either side
- [ ] I have searched the repository for any relevant issues or pull requests
- [ ] Any category I am creating has the minimum requirement of 3 items
- [ ] All changes have been [squashed][squash-link] into a single commit

[squash-link]: <https://github.com/todotxt/todo.txt-android/wiki/Squash-All-Commits-Related-to-a-Single-Issue-into-a-Single-Commit>

### CODE REVIEW

**Comment by micro-chipset** [comment]
Fix https://github.com/public-apis/public-apis/pull/1481

**Comment by micro-chipset** [comment]
Hello, @matheusfelipeog.
I have raised a PR against the label 'hacktoberfest' , waiting for feedback against the PR.

File: README.md [hunk 1]
```diff
@@ -333,6 +333,7 @@ API | Description | Auth | HTTPS | CORS |
 | [Frankfurter](https://www.frankfurter.app/docs) | Exchange rates, currency conversion and time series | No | Yes | Yes |
 | [National Bank of Poland](http://api.nbp.pl/en.html) | A collection of currency exchange rates (data in XML and JSON) | No | Yes | Yes |
 | [VATComply.com](https://www.vatcomply.com/documentation) | Exchange rates, geolocation and VAT number validation | No | Yes | Yes |
+| [Rwanda Locations](https://rapidapi.com/victorkarangwa4/api/rwanda) | Rwanda Provences, Districts,Cities,Capital City,Sector, cells, villages and streets | No | Yes | Unknown |
```
Comment (matheusfelipeog) [inline README.md:336]: Fix the spaces between `,`

Fix the alphabetical order too, please

```suggestion
| [Rwanda Locations](https://rapidapi.com/victorkarangwa4/api/rwanda) | Rwanda Provences, Districts, Cities, Capital City, Sector, cells, villages and streets | No | Yes | Unknown |
```

**Review by matheusfelipeog — CHANGES_REQUESTED** [review]
Thanks for your new contribution @micro-chipset ;)

Can you resolve this review, please?

File: README.md (hunk 1 above)
Comment (micro-chipset) [reply README.md]: @matheusfelipeog  fix spaces

File: README.md [hunk 2]
```diff
@@ -333,6 +333,7 @@ API | Description | Auth | HTTPS | CORS |
 | [Frankfurter](https://www.frankfurter.app/docs) | Exchange rates, currency conversion and time series | No | Yes | Yes |
 | [National Bank of Poland](http://api.nbp.pl/en.html) | A collection of currency exchange rates (data in XML and JSON) | No | Yes | Yes |
 | [VATComply.com](https://www.vatcomply.com/documentation) | Exchange rates, geolocation and VAT number validation | No | Yes | Yes |
+| [Rwanda Locations](https://rapidapi.com/victorkarangwa4/api/rwanda) | Rwanda Provences, Districts, Cities,Capital City, Sector, cells, villages and streets | No | Yes | Unknown |
```
Comment (matheusfelipeog) [inline README.md:336]: The correct alphabetical order is still missing (before VATComply.com). fix this, please.

```suggestion
| [Rwanda Locations](https://rapidapi.com/victorkarangwa4/api/rwanda) | Rwanda Provences, Districts, Cities, Capital City, Sector, cells, villages and streets | No | Yes | Unknown |
| [VATComply.com](https://www.vatcomply.com/documentation) | Exchange rates, geolocation and VAT number validation | No | Yes | Yes |
```

**Review by matheusfelipeog — CHANGES_REQUESTED** [review]
One thing is still missing, can you resolve this review?

File: README.md (hunk 2 above)
Comment (micro-chipset) [reply README.md]: fix correct alphabetical order

**Review by matheusfelipeog — APPROVED** [review]
Now it's Ok ✔

Thanks fo your update @micro-chipset ;)

**Pull request merged**